container/container.go

```diff
@@ -155,7 +155,7 @@ func (x *Container) readFromV2(m container.Container, checkFieldPresence bool) e
 
 		val = attrs[i].GetValue()
 		if val == "" {
-			return fmt.Errorf("empty attribute value %s", key)
+			return fmt.Errorf("empty value of the attribute %s", key)
```

Comment:
`empty %q attribute value` would be shorter.

Reply:
agree, changed